crates/header-translator/src/output.rs

```diff
@@ -39,6 +39,7 @@ impl Output {
         let mut macos_10_13_features: BTreeSet<String> = vec![
             "unstable-frameworks-macos-10-7".into(),
             "unstable-example-delegate".into(),
+            "unstable-example-metal".into(),
```

Comment:
To make CI pass, I think you need to add this to `icrate/Cargo.toml` as well (as the diff suggests).